types/egm96-universal/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for egm96-universal 1.0
+// Project: https://github.com/nicolas-van/egm96-universal
+// Definitions by: Muxable <https://github.com/muxfd>
```

Comment:
This should be your name (unless there's a reason otherwise)

Reply:
Done.